parachain/pallets/basic-channel/src/outbound/test.rs

```diff
@@ -135,7 +135,8 @@ fn test_submit_exceeds_payload_limit() {
 		let who: &AccountId = &Keyring::Bob.into();
 
 		let max_payload_bytes = MaxMessagePayloadSize::get();
-		let payload: Vec<u8> = (0..).take(max_payload_bytes as usize + 1).collect();
+
+		let payload: Vec<u8> = vec![1u8; (max_payload_bytes as usize + 1)].try_into().unwrap();
```

Comment:
What's the reason for this change? Are we just replacing `0, 1, 2, ...` with `1, 1, 1, ...`?

Reply:
because we change `MaxMessagePayloadSize` to 256 which does not apply to u8 if we still sample data like this. But I did  a little tweak to work as previously and also make test passed